extensions/tasks/NLUTestV0/index.ts

```diff
@@ -0,0 +1,109 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as tr from 'azure-pipelines-task-lib/toolrunner';
+import * as path from 'path';
+import * as utilities from 'nlu-devops-common/utilities';
+
+async function run() {
+    try {
+        const output = getOutputPath();
+        await runNLUTest(output);
+        await runNLUCompare(output);
+    } catch (error) {
+        tl.setResult(tl.TaskResult.Failed, (<Error>error).message);
+    }
+}
+
+async function runNLUTest(output): Promise<any> {
+    const tool = await utilities.getNLUToolRunner();
+    tool.arg('test')
+        .arg('-s')
+        .arg(tl.getInput('service'))
+        .arg('-u')
+        .arg(tl.getInput('utterances'))
+        .arg('-o')
+        .arg(output);
+
+    const serviceSettings = tl.getInput('serviceSettings')
```

Comment:
Once 0.2.5 is published, we'll need to change this to 'modelSettings' and use the '-m' flag below.